bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/styles.scss

```diff
@@ -1,11 +1,29 @@
-@import "../../../stylesheets/variables/_all";
-@import "../../../stylesheets/mixins/_scrollable";
+@import "/imports/ui/stylesheets/variables/_all";
+@import "/imports/ui/stylesheets/mixins/_scrollable";
+
+:root {
+  --uploadIconSize: 2.286rem;
```

Comment:
Where did you get these magic numbers from? Some of them look quite odd. Is it justified that some height/width values are px not rem? Also you have `  font-size: 117%;` - how are you getting these?